src/main/java/com/wnis/linkyway/dto/member/UpdateMemberRequest.java

```diff
@@ -0,0 +1,21 @@
+package com.wnis.linkyway.dto.member;
+
+import com.wnis.linkyway.validation.ValidationGroup;
+import lombok.*;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Pattern;
+
+
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+public class UpdateMemberRequest {
+    
+    @NotBlank(message = "닉네임을 입력해주세요", groups = ValidationGroup.NotBlankGroup.class)
+    @Pattern(regexp = "^[ㄱ-ㅎ가-힣a-zA-Z\\d_]{2,10}$",
```

Comment:
```suggestion
    @Pattern(regexp = "^[가-힣a-zA-Z\\d_]{2,10}$",
```

저번에 말씀드린다는걸 깜빡했는데 닉네임의 경우 한글 자음만 사용하게는 못하게 하는게 나을 것 같은데 어떻게 생각하시나요?

Reply:
오.... 좋은 것 같습니다. 이 부분은 제가 간과한 것 같네요... 다른 부분도 이런 식으로 적용된 코드가 있나 살펴봐야겠네요.. 감사합니다.